tests/e2e/specs/setup-mc/step-4-complete-campaign.test.js

```diff
@@ -234,6 +234,54 @@ test.describe( 'Complete your campaign', () => {
 				} );
 			} );
 
+			test.describe( 'Validate budget percent', () => {
```

Comment:
Could it use the same currency within this test group?

https://github.com/woocommerce/google-listings-and-ads/blob/99a5e2c75dedc2d9ab1032b9294177c9a66066b8/tests/e2e/specs/setup-mc/step-4-complete-campaign.test.js#L240

https://github.com/woocommerce/google-listings-and-ads/blob/99a5e2c75dedc2d9ab1032b9294177c9a66066b8/tests/e2e/specs/setup-mc/step-4-complete-campaign.test.js#L260

Reply:
👍